setup.cfg

```diff
@@ -6,3 +6,5 @@ tag = True
 [bumpversion:file:setup.py]
 search = VERSION = "{current_version}"
 
+[bumpversion:file:d3a_api_client/__init__.py]
+search = __version__ = "{current_version}"
```

Comment:
Do you think it would be possible to just use the `__version__` variable you added in `d3a_api_client/__init__.py`, and reference it in `setup.py`? In that way we would have only one variable for it (and bumpversion would just update that one).

Reply:
Another possible way to go would be to create a `VERSION` file and read it from `setup.py` (as described in this answer on SO: https://stackoverflow.com/a/2073599/1916588).

Reply:
@fievelk seems okay for me

Reply:
I'll take the first proposal as we're using the __version__ in the __init__ of d3a-web as well